lib/services/block.js

```diff
@@ -251,6 +253,7 @@ BlockService.prototype.confirm = function(block, ops) {
       //console.log(4);
       self._setBlockByTs(ops, block);
 
+      //console.log(4.1);
```

Comment:
Yes, I'll keep those for now, it's helpful for optimizations (will remove before release)